components/ide/code-desktop/supervisor-ide-config_insiders.json

```diff
@@ -1,6 +1,6 @@
 {
     "entrypoint": "/ide-desktop/startup.sh",
-    "entrypointArgs": [ "Open in VS Code Insiders on Desktop", "vscode-insiders" ],
+    "entrypointArgs": [ "{DESKTOPIDEPORT}", "Open in VS Code Insiders on Desktop", "vscode-insiders" ],
```

Comment:
We should remove 24000 from readinessProbe as well? it should be 24000 always, path should be enough.